src/executePrettier/buildPrettierOptions.js

```diff
@@ -10,11 +12,21 @@ const {
   isCurrentScopeVueScope,
 } = require('../editorInterface');
 const { shouldUseEditorConfig, getPrettierOptions, getAtomTabLength } = require('../atomInterface');
-const buildEditorConfigOptions = require('./buildEditorConfigOptions');
 const { getPrettierInstance } = require('../helpers');
 
+type EditorconfigToPretterResult = {
```

Comment:
`EditorConfigToPretterResult`

Reply:
haha I was thinking about that a while when I typed it actually.  I kept finding `const editorconfig = require('editorconfig')` as opposed to `const editorConfig = require('editorconfig');` but I'll change it.